README.md

```diff
@@ -129,7 +129,7 @@ docker-compose run hugo bash -l
 ```
 
 # Local development without using Docker
-To test without docker, you must have Hugo and node.js installed. You should install the latest HUGO and the latest LTS for node, 
+To test without docker, you must have Hugo, Go, and node.js installed. You should install the latest HUGO and the latest LTS for node, 
```

Comment:
Go is not required to be installed if you install a Hugo binary appropriate for your operating system. It is only needed if you are build Hugo from source. Perhaps we should make this clear or add a link to https://gohugo.io/getting-started/installing/.